petab/v2/problem.py

```diff
@@ -726,24 +727,29 @@ def get_optimization_to_simulation_parameter_mapping(self, **kwargs):
             )
         )
 
-    def sample_parameter_startpoints(self, n_starts: int = 100, **kwargs):
-        """Create 2D array with starting points for optimization
+    def get_priors(self) -> dict[str, Distribution]:
+        """Get prior distributions.
 
-        See :py:func:`petab.sample_parameter_startpoints`.
+        :returns: The prior distributions for the estimated parameters.
         """
-        return sampling.sample_parameter_startpoints(
-            self.parameter_df, n_starts=n_starts, **kwargs
-        )
+        return {
+            p.id: p.prior_dist
+            for p in self.parameter_table.parameters
+            if p.estimate
```

Comment:
Seems reasonable to drop non-estimated parameter priors, but then users might see unexpected posterior values. e.g. if they perform MAP then fix the parameters to those values and recompute, there will be an undocumented change in the posterior value because their `estimate` column has changed.
So perhaps priors + fixed parameters should be an error or warning?

Reply:
Hm, I am not sure. It's effectively a different problem then (kind of like changing your prior to a Dirac delta?), so that a change in the posterior should be expected, isn't it?

Reply:
Agreed, a change can be expected, but then priors should be treated as an error since they are dropped, or?

Reply:
Yes, same issue there, but the worst thing that happens there is that the user looks into their parameter estimates and sees that they can't find the parameter that they thought they estimated. In the priors case, it would be difficult to notice the mistake.

But you're right, and since it's nice to toggle parameters on or off without having to change other columns, fine to not have an error. As you like, but a warning/info/debug for unused priors could still be nice

Reply:
That could be part of some (yet to be implemented) warning mode for validation then, but I'd still find it somewhat confusing to treat priors different than bounds in that situation. Maybe that's something to leave to the parameter estimation tools.